readme.md

```diff
@@ -8,7 +8,7 @@ For the following quiz, there will be a series of questions or directions follow
 What is Git, what problem does it solve?
 
 ```
-answer goes here
+Git is a version control tool used to share and track multiple instances of the same file. Without Git, the problem would be that you would have way too many copies of the same exact file. Without the Git technology, each version of the file you save would have to be an exact duplicate of the previous version, so for instance it would look something like this: fileV1 > fileV2 > fileV3 > fileV4final_copy.
 
 ```
 
```

Comment:
It's not that you'd have too many copies of the same file, Git is what allows you to view the same file in multiple states but that doesn't impact how many copies of a file there would be.